Data Deletion Request Framework.md

```diff
@@ -120,38 +125,28 @@
 
 <ol>
   <li><span style="color:rgb(255, 0, 0);">
-    <strong>*Data Subject</strong>
-    requests a data deletion from a 1st Party*</span></li>
+    <strong>*Data Subject</strong> requests a data deletion from a 1st Party*</span></li>
   <ol>
     <li>1st Party validates the request.</li>
     <li>1st Party determines what identifiers are subject to the request.</li>
     <li>1st Party determines partners that data has been shared with which the request must be communicated to.</li>
     <li>1st party accesses the dsrdelete.json resource of each identified partner to determine how IDs must be formatted for them.&nbsp;</li>
-    <li>1st party generates an initial idJWT for each identifier that is subject to the request.</li>
+    <li>1st party generates an initial orJWT for each identifier that is subject to the request.</li>
```

Comment:
Since the orJWT no longer includes an identifier, the 1st party needs only generate one for each user request, not one for each identifier. Suggested edit:

> <li>1st party generates an orJWT for the request.</li>